### ISSUE TITLE
New install on Mac M3Max - gitlab runner fails.

### ISSUE THREAD

**Issue by paxinos** [kubefirst/kubefirst#2187]
### Which version of kubefirst are you using?

2.4.8

### Which cloud provider?

k3d (local)

### Which DNS?

None specific

### Which installation type?

CLI

### Which distributed Git provider?

GitLab

### Did you use a fork of `gitops-template`?

No

### Which Operating System?

macOS

### What is the issue?

Tried a couple times with the same results.   

Command line is:

 kubefirst k3d create --git-provider gitlab --gitlab-group theosis1 --cluster-name theosis1

Verfied gitlab-token is valid and has proper permissions.

Another Mac issue is when using bash as the shell, if kubefirst exits I need to run 'stty sane' or else the shell behaves poorly (doesn't use newlines/returns,... doesn't show cursor,....  )

```
Registration attempt 28 of 30                                                                                                                                   │
│ Runtime platform                                    arch=arm64 os=linux pid=480 revision=d540b510 version=15.9.1                                                │
│ WARNING: Running in user-mode.                                                                                                                                  │
│ WARNING: The user-mode requires you to manually start builds processing:                                                                                        │
│ WARNING: $ gitlab-runner run                                                                                                                                    │
│ WARNING: Use sudo for system-mode:                                                                                                                              │
│ WARNING: $ sudo gitlab-runner...                                                                                                                                │
│                                                                                                                                                                 │
│ Merging configuration from template file "/configmaps/config.template.toml"                                                                                     │
│ PANIC: The registration-token needs to be entered                                                                                                               │
│ Registration attempt 29 of 30                        
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fharper** [comment]
Thanks for reporting the issues @paxinos .

For stty, it's probably related to https://github.com/kubefirst/kubefirst/issues/2176 . You can check the comment, and try with a PR from our team. I wasn't able to test it yet, but it should fix the problem.

As for the GitLab issue, can you share the error from the GitLab Runner please?

**Comment by paxinos** [comment]
> Thanks for reporting the issues @paxinos .
> 
> For stty, it's probably related to #2176 . You can check the comment, and try with a PR from our team. I wasn't able to test it yet, but it should fix the problem.

Thanks. Will try.

> As for the GitLab issue, can you share the error from the GitLab Runner please?

The only error I have is what is in the initial report.   It is repeated 30 times and then the container fails.

The key error seems to be:

     PANIC: The registration-token needs to be entered                                                                                                               │

**Comment by paxinos** [comment]
Thanks @johndietz !   Appreciated!    Glad to know it wasn't me ;-)

**Comment by CristhianF7** [comment]
Hey @paxinos, thanks for reporting this issue.

I can confirm this has been fixed in v2.4.12.

**Comment by fharper** [comment]
Oops, we were too fast to celebrate this issue fixed. The new CLI isn't available on Homebrew yet: https://github.com/Homebrew/homebrew-core/pull/177411
